database/migrations/2021_03_29_191548_create_keys_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateKeysTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('keys', function (Blueprint $table) {
+            $table->id();
+            $table->string('value');
+            $table->unsignedBigInteger('game_id');
```

Comment:
Ik zou geen surrogaat sleutel gebruiken hier. De value van de key en game_id zijn samen als het goed is al uniek. Een extra onbekend id toevoegen maakt het volgens mij alleen maar onhandiger.